CMakeLists.txt

```diff
@@ -190,6 +194,7 @@ install(TARGETS async_slam_toolbox_node
                 sync_slam_toolbox_node
                 localization_slam_toolbox_node
                 map_and_localization_slam_toolbox_node
+                # lifelong_slam_toolbox_node
```

Comment:
commented out?

Reply:
Originally, the lifelong_slam_toolbox_node did not exist in the `install`.  
I thought this was intentional since it is experimental. 
I commented out and added the lifelong_slam_toolbox_node , should I remove `#` ?

Reply:
Yes

Reply:
[bac68a7767e6b79a1eb6208174a1e890cb0c7246]
removed.